sstables_loader.cc

```diff
@@ -488,8 +491,25 @@ class sstables_loader::download_task_impl : public tasks::task_manager::task::im
     virtual tasks::is_internal is_internal() const noexcept override {
         return tasks::is_internal::no;
     }
+
+    tasks::is_abortable is_abortable() const noexcept override {
+        return tasks::is_abortable::yes;
+    }
+    void abort() noexcept override;
 };
 
+void sstables_loader::download_task_impl::abort() noexcept {
+    if (_as.abort_requested()) {
+        return;
+    }
+    impl::abort();
```

Comment:
I wonder about the order.
My intuition is to first request abort and then call the upstream abort method.
If the order matters, please document why.

Reply:
It does. Will add comment.